lib/utf8.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports.decode = function (input) {
+  if (Buffer.isBuffer(input) === false) {
```

Comment:
This function is overloaded. Sometimes it takes a buffer, and sometimes it takes an array of Numbers. That is confusing, as they represent different things.